controllers/configuration_controller.go

```diff
@@ -182,9 +182,10 @@ func (r *ConfigurationReconciler) Reconcile(
 	// check dependencies are ready
 	ready, err := r.checkReadiness(ctx, obj.GetNamespace(), &obj.Spec.SourceRef)
 	if err != nil {
-		status.MarkNotReady(r.EventRecorder, obj, v1alpha1.SourceRefNotReadyWithErrorReason, err.Error())
+		status.MarkUnknown(r.EventRecorder, obj, v1alpha1.SourceRefNotReadyWithErrorReason, err.Error())
 
-		return ctrl.Result{RequeueAfter: obj.GetRequeueAfter()}, nil
+		// we are watching the source object which should re-trigger the reconcile loop
+		return ctrl.Result{}, nil
```

Comment:
are we just leaving the default value? Why aren't we using a defined `interval`?

Reply:
Because we have a Watch on the source, so there is no reason to Requeue.